sys/net/application_layer/nanocoap/nanocoap.c

```diff
@@ -106,6 +106,11 @@ int coap_parse(coap_pkt_t *pkt, uint8_t *buf, size_t len)
             DEBUG("option count=%u nr=%u len=%i\n", option_count, option_nr, option_len);
 
             if (option_delta) {
+                if (option_count >= NANOCOAP_NOPTS_MAX) {
+                    DEBUG("nanocoap: max nr of options exceeded");
```

Comment:
I agree debug statement is helpful. Needs newline.

Reply:
> Needs newline.

fixed